ui/src/main/webapp/src/app/components/project-list/project-list.component.ts

```diff
@@ -56,12 +54,6 @@ export class ProjectListComponent implements OnInit, OnDestroy {
     ngOnInit():any {
         this.updateSort();
         this.getMigrationProjects();
-        this._refreshIntervalID = setInterval(() => this.getMigrationProjects(), 30000);
```

Comment:
@klinki - I agree with you, I just don't want to block this PR if this is the only significant unrelated change.